test/DoctrineBatchUtilsTest/BatchProcessing/SelectBatchIteratorAggregateTest.php

```diff
@@ -180,7 +180,7 @@ private function assertSuccessfulReFetchesInNestedIterableResult(SelectBatchIter
                 ['Yadda', ['id' => 456], $freshObjects[1]],
             ]
         );
-        $this->entityManager->expects(self::at(2))->method('clear');
+        $this->entityManager->expects(self::once())->method('clear');
```

Comment:
Goes back to https://github.com/Ocramius/DoctrineBatchUtils/pull/96/files#r539520806

Reply:
see https://github.com/Ocramius/DoctrineBatchUtils/pull/96#discussion_r542267620